tests/src/Kernel/Migrate/OgFieldD7MigrateTest.php

```diff
@@ -44,23 +46,48 @@ class OgFieldD7MigrateTest extends MigrateDrupal7TestBase {
   protected function setUp() {
     parent::setUp();
 
+    $this->fileMigrationSetup();
     $this->loadFixture(__DIR__ . '/../../../fixtures/drupal7.php');
+
+    $this->installEntitySchema('node');
+    $this->installEntitySchema('comment');
+    $this->installEntitySchema('taxonomy_term');
     $this->installConfig(static::$modules);
+
     $this->createNodeCommentCombination('page');
     $this->createNodeCommentCombination('article');
     $this->createNodeCommentCombination('blog');
     $this->createNodeCommentCombination('book');
     $this->createNodeCommentCombination('forum', 'comment_forum');
     $this->createNodeCommentCombination('test_content_type');
+
     Vocabulary::create(['vid' => 'test_vocabulary'])->save();
+
     $this->executeMigrations([
       'd7_field',
       'd7_field_instance',
+      'd7_user_role',
+      'd7_user',
+      'd7_node_type',
+      'd7_node',
       'd7_og_group',
       'd7_og_field_instance',
     ]);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function getFileMigrationInfo()
+  {
```

Comment:
Left braces should be on the same line as the function declaration per Drupal coding standards.